src/components/common/DynamicTabs/useTabs.ts

```diff
@@ -212,38 +211,36 @@ export function useTabs(persistanceKey: string) {
         if (!name || !url || !kind) {
             return Promise.resolve(false)
         }
-        // @ts-expect-error available on all modern browsers
-        const { promise, resolve } = Promise.withResolvers<boolean>()
```

Comment:
Unsupported in recent versions of firefox and safari: https://caniuse.com/?search=withResolvers